nix/hls.nix

```
@@ -0,0 +1,22 @@
+{ pkgs
+, ghcVersion
+}:
+let
+  sources = import ./sources.nix { };
```

```suggestion
  inherit (pkgs) sources;
```
this should allows to remove nixpkgs entry from sources.json